src/BlazorWebView/src/Maui/Microsoft.AspNetCore.Components.WebView.Maui.csproj

```diff
@@ -3,11 +3,18 @@
   <PropertyGroup>
     <TargetFrameworks>$(_MauiDotNetTfm);$(MauiPlatforms)</TargetFrameworks>
     <Nullable>enable</Nullable>
-    <Description>Build .NET MAUI applications with Blazor web UI in the BlazorWebView control.</Description>
     <DefineConstants>$(DefineConstants);WEBVIEW2_MAUI</DefineConstants>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- NuGet package information -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.AspNetCore.Components.WebView.Maui</PackageId>
+    <PackageTags>$(DefaultPackageTags);blazor;webview;aspnet</PackageTags>
+    <Description>Build .NET MAUI applications with Blazor web UI in the BlazorWebView control.</Description>
+  </PropertyGroup>
```

Comment:
Looks like the AspNetCore packages all use the purple square .NET icon. I can't find where in the build it comes from, but you can go to https://nuget.info/packages/Microsoft.AspNetCore.Components.WebView/8.0.0-rc.2.23480.2 and download Icon.png, add to this repo, and reference that in the metadata here.